src/demandlib/vdi/regions.py

```diff
@@ -0,0 +1,559 @@
+"""
+The region module combines profiles from VDI 4655 for heat and power demand of
+houses within one region.
+
+
+Module VDI 4655
+---------------
+This is an implementation of the calculation of economic efficiency
+using the annuity method defined in the German VDI 4655.
+
+    **VDI 4655**
+
+    **Reference load profiles of single-family and
+    multi-family houses for the use of CHP systems**
+
+    *May 2008 (ICS 91.140.01)*
+
+Copyright:
+
+    *Verein Deutscher Ingenieure e.V.*
+
+    *VDI Standards Department*
+
+    *VDI-Platz 1, 40468 Duesseldorf, Germany*
+
+Reproduced with the permission of the Verein Deutscher Ingenieure e.V.,
+for non-commercial use only.
```

Comment:
I fear we may not implement/use this for an open source project. I will stop my review here.

Reply:
Hello there,
in 2019 I asked for permission from the VDI. I explicitly mentioned to the VDI that the implementations of VDI norms would become public code on GitHub as part the work done for scientific projects. I received that permission via email with the only prerequisite being the reference to the VDI norms and the note „Wiedergegeben mit Erlaubnis des Verein Deutscher Ingenieure e. V.“ Afterwards I published my code in "lpagg", which uvchik now adapted for "demandlib". If that helps, I could forward you the full email.

Reply:
If the implementation contains code provided by the VDI, the VDI has to agree to a license. If the code was not provided by the VDI, I doubt that it has copyright. (At least if the code is an implementation and not "just" a derived work.)

If using _the implementation_ is really only allowed for non-commercial users, that is contradicting every possible open source license. (Plus: Non-commercial is not adequately defined. If your institution has third-party funded projects, it can be considered commercial. Just one of the many reasons, why there is no open source just for "non commercial".)

Reply:
As we do have the permission to write public code using the calculations and values of VDI 4655 we can continue.

I remove the "for non-commercial use only" because it is not in line with the used MIT license. Furthermore I remove the copyright in the reference to make sure that it is a reference and not the copyright of the code.